src/main/java/org/jenkinsci/plugins/durabletask/FileMonitoringTask.java

```diff
@@ -103,8 +105,30 @@
      */
     private @CheckForNull String charset;
 
+    /**
+     * Provides the cookie value for a given {@link FilePath} workspace. It also uses
+     * a flag to identify if we want this cookie hash based on MD5 (former/old format) or SHA-256 algorithm.
+     * This method is only used to maintain backward compatibility on stopping tasks that were
+     * launched before the new format was applied.
+     * @param workspace - workspace path used to setup the digest
+     * @param old - boolean to select if we want to use former/old hash algorithm to maintain backward compatibility
+     * @return
```

Comment:
```suggestion
```

or actually provide content

Reply:
Updated to provide some content on https://github.com/jenkinsci/durable-task-plugin/pull/141/commits/930b0ab95b2736951a7baadbd8fb5961dc177176